source/includes/_dataset_security.md

```diff
@@ -0,0 +1,339 @@
+# Dataset security
+
+The dataset security is defined through 3 variables:
+
+* a general access policy
+* specific rulesets for users and groups
+* a default ruleset that applies to anybody else
+
+The general access policy sets whether the dataset is accessible to anyone or just a few specific users.
```

Comment:
Why define just one of the items of the list in its own paragraph? I understand that this one is less obvious than the other ones and might need an early explanation, but then maybe it would fit better inside the list itself (something like `* a general access policy, that sets whether the dataset is accessible to anyone or just a few specific users.`).